app/routes/public/sessions.js

```diff
@@ -140,7 +140,43 @@ export default class SessionsRoute extends Route {
         perPageParam : 'page[size]',
         pageParam    : 'page[number]',
         store        : eventDetails
-      })
+      }),
+      schedules: await this.getSchedules(eventDetails)
     };
   }
+
+  async getSchedules(event) {
+    const scheduledFilterOptions = [
+      {
+        and: [
+          {
+            name : 'starts-at',
+            op   : 'ne',
+            val  : null
+          },
+          {
+            or: [
+              {
+                name : 'state',
+                op   : 'eq',
+                val  : 'accepted'
+              },
+              {
+                name : 'state',
+                op   : 'eq',
+                val  : 'confirmed'
+              }
+            ]
+          }
+        ]
+      }
+    ];
+
+    const sessions = await event.query('sessions', {
+      filter       : scheduledFilterOptions,
+      'page[size]' : 0
```

Comment:
You are

Reply:
@iamareebjamal  sir by now the url for getting start-dates look-likes 

  http://localhost:5000/v1/events/4/sessions?fields[session]=starts-at&page[size]=0

if i will not use page[size]=0 , then how will i get all the start-dates